packs/feat-effects/effect-furnace-form.json

```
@@ -0,0 +1,85 @@
+{
+    "_id": "Tw9MjeQHL3qFY1PO",
+    "img": "systems/pf2e/icons/spells/fiery-body.webp",
+    "name": "Effect: Furnace Form",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Furnace Form]</p>\n<p>You become living flame, giving you fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water. Any creature that touches you or damages you with an unarmed attack or non-reach melee weapon takes @Damage[3d6[fire]] damage. Your unarmed attacks deal 1d4 additional fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses). Your fire Elemental Blasts deal an additional die of damage. In fire form, you have a fly Speed of 40 feet and don't need to breathe.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Furnace Form]</p>\n<p>You become living flame, giving you fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water. Your unarmed attacks deal 1d4 additional fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses). Your fire Elemental Blasts deal an additional die of damage. In fire form, you have a fly Speed of 40 feet.</p>"
```
Removing stuff the effect item doesn't automate.

---

The effect for fiery body has that text. Also even though it doesn't automate it, isn't it helpful information when someone mouses over the effect panel to see the description?